custom-prom.md

```diff
@@ -8,6 +8,10 @@ Custom Prometheus and Grafana
 
 When integrating Kubecost with an existing Prometheus, we recommend first installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)) as a dry run before integrating with an external Prometheus deployment. You can get in touch (support@kubecost.com) or via our [Slack community](https://join.slack.com/t/kubecost/shared_invite/enQtNTA2MjQ1NDUyODE5LWFjYzIzNWE4MDkzMmUyZGU4NjkwMzMyMjIyM2E0NGNmYjExZjBiNjk1YzY5ZDI0ZTNhZDg4NjlkMGRkYzFlZTU) for assistance.
 
+The Kubecost Prometheus deployment is used both as a source and a sink for cost & capacity metrics. It’s optimized to not interfere with other observability instrumentation and by default only contains metrics that are useful to the Kubecost product. This amounts to retaining __70-90% fewer metrics__ than a standard Prometheus deployment.
```

Comment:
> It’s optimized to not interfere with other observability instrumentation and by default only contains metrics that are useful to the Kubecost product.

Should we say "The default Kubecost Prometheus..." here to clarify the reference?

Reply:
Maybe this?
"This results in __70-90% fewer metrics__ than a Prometheus deployment using default settings."